lms/models/application_instance.py

```diff
@@ -92,7 +92,7 @@ class ApplicationInstance(CreatedUpdatedMixin, Base):
 
     consumer_key = sa.Column(sa.Unicode, unique=True, nullable=True)
     shared_secret = sa.Column(sa.Unicode, nullable=False)
-    lms_url = sa.Column(sa.Unicode(2048), nullable=False)
+    lms_url: Mapped[str] = mapped_column(sa.Unicode(2048), nullable=False)
```

Comment:
We could probably automate a conversion to this style for all models. Doing only the ones that solve issues in this PR.